cypress/e2e/ResponsiveDesign/is_responsive.js

```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+
+const { setViewPort } = require("../../support/functions");
+const { homePage, resolution } = require("../../support/selectors");
+
+describe('Make sure BBC.com loads properlly', () => {
+
+    it('Test Case 24. responsivness 1920x1080', () => {
+
+        setViewPort(resolution.PC1);
+        cy.get(homePage.navBar).should('be.visible');
+        cy.get(homePage.leftArticleBlock).should('be.visible');
+        cy.get(homePage.rightArticleBlock).should('be.visible');
+        cy.get(homePage.centralArticle).should('be.visible');
+        cy.get(homePage.signInButton).eq(0).should('be.visible');
+        cy.get(homePage.registerButton).eq(0).should('be.visible');
```

Comment:
bet ar be.visible sitas assert tikrai patikrina ar responsive page yra ? Be to galejai visa sita asserta issikelt i funkcija atsikira ir apsirasyt ja kaip parametra pasiduot nustatyta viewporta ir assertint negu visus kartus persirasineti ant kiekvieno screeno tuos pacius stepsus

Reply:
galima butu sudet i funkcijas bet tik to pacio tipo irenginius nes kompams, plancetem ir telefonams ne tik skiriasi parametrai bet ir assertai. Galima dali kur tik parametrai skiriasi isskirti i atskiras funkcijas ir kurios skiriasi atskirai prirasyti bet klausimas ar iseitu svariau